model/users.js

```diff
@@ -31,5 +31,16 @@ class User {
       email: result.rows[0].email,
     };
   }
+
+  static async getUser(userId) {
+    try {
+      const result = await connection.pool.query(
+        `SELECT * FROM users WHERE id =${userId}`
```

Comment:
Would it not be better to put `SELECT user_name FROM users WHERE user_id=${user_id};`?
So you don't have to select which column you want as it would return the exact username?

Reply:
you are right - good spot.